docs/classic-ui/forms.md

```diff
@@ -120,6 +120,324 @@ schema = IMyForm
+            )
+```
+
+In supermodel XML, this can be specified as:
+
+```xml
+    <field type="zope.schema.TextLine"
+           name="dummy"
+           form:omitted="true">
+        <title>Dummy</title>
+    </field>
+
+    <field type="zope.schema.TextLine"
+           name="edit-only"
+           form:omitted="z3c.form.interfaces.IForm:true z3c.form.interfaces.IEditForm:false">
+        <title>Only included on edit form</title>
+    </field>
+```
+
+`form:omitted` may be either a single boolean value, or a space-separated
+list of form_interface:boolean pairs.
+
+
+### Re-ordering fields
+
+A field's position in the form can be influenced using the `order_before`
+and `order_after` directives. In this example, the `not_last` field
+is placed before the `summary` field even though it is defined afterward:
+
+```python
+    from plone.supermodel import model
+    from plone.autoform import directives as form
+
+    class IMySchema(model.Schema):
+
+        summary = schema.Text(
+            title=u"Summary",
+            description=u"Summary of the body",
+            readonly=True
+            )
+
+        form.order_before(not_last='summary')
+        not_last = schema.TextLine(
+            title=u"Not last",
+            )
+```
+
+The value passed to the directive may be either '*' (indicating before or after
+all fields) or the name of another field. Use `'.fieldname'` to refer to
+field in the current schema or a base schema. Prefix with the schema name (e.g.
+`'IDublinCore.title'`) to refer to a field in another schema. Use an
+unprefixed name to refer to a field in the current or the default schema for
+the form.
+
+In supermodel XML, the directives are called `form:before` and `form:after`.
+For example:
+
+```xml
+    <field type="zope.schema.TextLine"
+           name="not_last"
+           form:before="*">
+        <title>Not last</title>
+    </field>
+```
+
+
+### Organizing fields into fieldsets
+
+Fields can be grouped into fieldsets, which will be rendered within an HTML
+`<fieldset>` tag. In this example the `footer` and `dummy` fields
+are placed within the `extra` fieldset:
+
+```python
+    from plone.supermodel import model
+    from plone.autoform import directives as form
+
+    class IMySchema(model.Schema):
+
+        model.fieldset('extra',
+            label=u"Extra info",
+            fields=['footer', 'dummy']
+            )
+
+        footer = schema.Text(
+            title=u"Footer text",
+            )
+
+        dummy = schema.Text(
+            title=u"Dummy"
+            )
+```
+
+In supermodel XML fieldsets are specified by grouping fields within a
+`<fieldset>` tag:
+
+```xml
+  <fieldset name="extra" label="Extra info">
+      <field name="footer" type="zope.schema.TextLine">
+          <title>Footer text</title>
+      </field>
+      <field name="dummy" type="zope.schema.TextLine">
+          <title>Dummy</title>
+      </field>
+  </fieldset>
+```
+
+
+### Changing a field's widget
```

Comment:
this is already covert in the widgets chapter and fits better there.
In general, we should try to place individual parts of autoform docs into there fitting chapters.
Could be anything from schemas, fields, widgets to forms and content types.
As content types and forms are both sharing schemas, fields and widgets, I would first try to fit in one of those and reference it from forms and content types where it make sense.

Reply:
I agree with @MrTango. This section already exists in https://6.docs.plone.org/backend/widgets.html#changing-a-field-s-widget. I won't review its content, as the existing content has already been reviewed and deployed.